packages/tackle-box/lib/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box } from "@/components/Box/Box";
+import { HStack } from "@/components/Stack/Stack";
+import { createContext, ReactNode, useContext, useState } from "react";
+import { css, html } from "react-strict-dom";
+import { Text } from "@/components/Text/Text";
+
+const TabsContext = createContext<{
+  activeTab: string;
+  setActiveTab: (tab: string) => void;
+}>({
+  activeTab: "",
+  setActiveTab: () => {},
+});
+
+function useTabsContext() {
+  return useContext(TabsContext);
+}
+
+type RootProps = {
+  children: ReactNode;
+  defaultValue?: string;
+};
+
+function Root({ children, defaultValue }: RootProps) {
+  const [activeTab, setActiveTab] = useState(defaultValue ?? "");
+
+  return (
+    <TabsContext.Provider value={{ activeTab, setActiveTab }}>
+      {children}
+    </TabsContext.Provider>
+  );
+}
+
+type ListProps = {
+  children: ReactNode;
+};
+
+function List({ children }: ListProps) {
+  return (
+    <HStack justifyContent="center" borderBottomWidth={1} borderColor="divider">
+      {children}
```

Comment:
If we want to make things more accessible, we should have the children here be a `ul` and then we can wrap the trigger elements in an `li`.

The `ul` can then also have the role of `tablist` and each `li` can have a role of `tab`